src/renderer/components/composer/menuAttachment.tsx

```diff
@@ -101,6 +105,36 @@ export default function MenuAttachment({
     tx,
   ])
 
+  const selectContact = async () => {
+    const { setLastPath } = rememberLastUsedPath(LastUsedSlot.Attachment)
+    let dialogId = ''
+    /**
+     * TODO: reduce the overhead: just provide a vcardContact to draft/message
```

Comment:
Maybe add to core a function like `miscSetDraft()` but taking an array of contact ids instead of file path? Say, `setDraftVcard()`?

Reply:
I'd imagine more like a `setVcard(draft_msgid, contactId/s)` and doing that when we do the new apis during the upcoming composer rewrite, so not in this version.

Reply:
Anyway i've created a PR for it, so you can check if the API fits the needs: https://github.com/deltachat/deltachat-core-rust/pull/5677